testing/test_reports.py

```diff
@@ -396,6 +396,18 @@ def check_longrepr(longrepr):
         # for same reasons as previous test, ensure we don't blow up here
         loaded_report.longrepr.toterminal(tw_mock)
 
+    def test_report_prevent_ConftestImportFailure_hiding_exception(self, testdir):
+        sub_dir = testdir.tmpdir.join("ns").ensure_dir()
+        sub_dir.join("conftest").new(ext=".py").write("import unknown")
+
+        result = testdir.runpytest_subprocess(".")
+        result.stdout.fnmatch_lines(
+            ["E   ModuleNotFoundError: No module named 'unknown'"]
+        )
+        result.stdout.no_fnmatch_line(
```

Comment:
To make this more robust, I'd match just `*ConftestImportFailure*`.

Reply:
I needed to include `ERROR  - *ConftestImportFailure*` as otherwise tests including `ConftestImportFailure` in their name would fail.